.ci/azure-pipelines/env.yml

```diff
@@ -74,9 +67,8 @@ jobs:
       command: build
       arguments: |
         --no-cache
-        --build-arg CUDA_VERSION=$(CUDA_VERSION)
-        --build-arg UBUNTU_DISTRO=$(UBUNTU_DISTRO)
-        --build-arg USE_CUDA=$(USE_CUDA)
+        --build-arg UBUNTU_VERSION=$(UBUNTU_VERSION)
+        --build-arg USE_LATEST_CMAKE=$(USE_LATEST_CMAKE)
```

Comment:
Is it not necessary to also specify `ENSENSOSDK_VERSION` here?

Reply:
No, when nothing is specified, the default value will be used. So I specified it for the last 2.x release, but for 3.x I use the default value specified in the Dockerfile.

Reply:
I mean: You set `ENSENSOSDK_VERSION: 2.3.1557` for Ubuntu 18.04 in line 49. Does that value get passed correctly to the Dockerfile if there isn't something like `--build-arg ENSENSOSDK_VERSION=$(ENSENSOSDK_VERSION)`? As there is for `UBUNTU_VERSION`, `USE_LATEST_CMAKE`, and `VTK_VERSION`?

Reply:
Ah yes, saw your comment just now, but saw this during rebasing, too 😆